src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+spring:
+  jpa:
+    open-in-view: false
+    show-sql: true
+  datasource:
+    username: baeldung
```

Comment:
Oh my, know I know who is actually Baeldung 😱

Reply:
too lazy to update the creds